aea/protocols/base.py

```diff
@@ -56,23 +56,42 @@ def __str__(self):
             """Get the string representation."""
             return str(self.value)
 
+    class _SlotsCls:  # pylint: disable=too-few-public-methods
+        __slots__: Tuple[str, ...] = (
+            "performative",
+            "dialogue_reference",
+            "message_id",
+            "target",
+            "content",
```

Comment:
Not sure why we allow for the field `content` here. We don't want anyone to use the Message class directly. It should probably be made ABC...thoughts @marcofavorito ?

Reply:
@DavidMinarsch 
it was used in some test cases, anyway it's redefined for every Message class

Reply:
So the test cases are wrong imo. The Message class is not to be used as it does not reside in a complete protocol.

Reply:
> Not sure why we allow for the field `content` here. We don't want anyone to use the Message class directly. It should probably be made ABC...thoughts @marcofavorito ?

I agree

Reply:
@solarw please fix

Reply:
@DavidMinarsch @marcofavorito 
I found that the class being tested is not used anywhere instead of this test, so it moved to test to cover base serializer.